charts/pathfinder-rpc-server/templates/pathfinder2-updater-statefulset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  annotations:
+    kompose.cmd: kompose convert --controller statefulset -o pathfinder-rpc-server -c
+    kompose.version: 1.26.1 (a9d05d509)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: pathfinder2-updater
+  name: pathfinder2-updater
+  namespace: pathfinder
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: pathfinder2-updater
+  serviceName: pathfinder2-updater
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: pathfinder2-updater
+    spec:
+      containers:
+        - env:
+            - name:     EXTERNAL_CAPACITY_GRAPH_PATH
+              valueFrom:
+                configMapKeyRef:
+                  key:  PATHFINDER_UPDATER_EXTERNAL_CAPACITY_GRAPH_PATH
+                  name: env
+            - name:     INDEXER_DB_CONNECTION_STRING
+              valueFrom:
+                secretKeyRef:
+                  key:  PATHFINDER_UPDATER_INDEXER_DB_CONNECTION_STRING
+                  name: pathfinder-updater-connection-string 
+            - name:     INDEXER_WS_URL
+              valueFrom:
+                configMapKeyRef:
+                  key:  PATHFINDER_UPDATER_INDEXER_WS_URL
+                  name: env
+            - name:     INTERNAL_CAPACITY_GRAPH_PATH
+              valueFrom:
+                configMapKeyRef:
+                  key:  PATHFINDER_UPDATER_INTERNAL_CAPACITY_GRAPH_PATH
+                  name: env
+            - name:     PATHFINDER_RPC_URL
+              valueFrom:
+                configMapKeyRef:
+                  key:  PATHFINDER_UPDATER_RPC_URL
+                  name: env
+          image: circlesubi/pathfinder2-updater:latest
+          name: pathfinder2-updater
+          resources: {}
+          volumeMounts:
+            - mountPath: /var/pathfinder2/data
+              name: pathfinder2-claim0
+      volumes: 
+        - name: pathfinder2-claim0 
+          persistentVolumeClaim:
+            claimName: pathfinder2-claim0-pathfinder2-0
```

Comment:
Would that work?

```suggestion
            claimName: pathfinder2-0
```

Reply:
yes, actually the two statefulset remain separate and one of the statefulset claims the pvc from the other statefulset

Reply:
This answer seems to respond to https://github.com/CirclesUBI/circles-helm/pull/17#pullrequestreview-1479713616 and I would like to pick it up from there, too.

What did you think about my suggestion to simplify the name of the pathfinder2 PVC?

I think what I wrote above is not perfectly accurate, as I don't know what the actual Claim name would be, but I like less "technical" naming at times. Here we could remove some redundancy: It's the only claim we make, considered the Indexer is not stateful, and we don't need to double the term pathfinder2.

Just an idea.

Reply:
renamed